images/capi/packer/files/bootstrap-flatcar.sh

```diff
@@ -15,10 +16,19 @@ fi
 
 PYPY_VERSION=7.0.0
 
+wget -O - https://bitbucket.org/squeaky/portable-pypy/downloads/pypy-$PYPY_VERSION-linux_x86_64-portable.tar.bz2 | tar -xjf -
+mv -n pypy-$PYPY_VERSION-linux_x86_64-portable pypy2
+ln -s ./pypy2/bin/pypy python
```

Comment:
not `python2`?

Reply:
Added a symlink `python2` as well.

I have also tried removing `/opt/bin/python`, but it does not work.
Main reason is because `/opt/bin/pypy2/bin/virtualenv-pypy` simply requires `/opt/bin/python`, not `/opt/bin/python2`.
So I had to keep `/opt/bin/python`.